config/endpoints/14_dgfip_attestation_fiscale.yml

```diff
@@ -6,8 +6,8 @@
     entity_type_description: |+
       Cette API délivre uniquement les attestations fiscales des **✅ entreprises soumises à l'impôt sur les sociétés (IS)**. 
 
-      L'API ne permet pas de récupérer les attestations fiscales : 
-      - ❌ des entreprises qui relèvent de l'impôt sur le revenu (IR), les entreprises individuelles sont donc notamment exclues ;
+      **L'API ne permet pas de récupérer les attestations fiscales** : 
+      - ❌ des entreprises qui relèvent de l'impôt sur le revenu (IR), les entreprises individuelles et micro-entreprises sont donc exclues. _Les entreprises qui relèvent des BIC, BNC ou BA, bénéfices respectivement industriels ou commerciaux, non-commerciaux ou agricoles, sont imposées à l'impôt sur le revenu. Leur attestation fiscale n'est donc pas récupérable par cette API_ ;
```

Comment:
```suggestion
      - ❌ des entreprises qui relèvent de l'impôt sur le revenu (IR), quelque soit leur régime d'imposition, BIC, BNC ou BA. **À partir du moment où l'entreprise est imposée à l'IR, son attestation fiscale ne peut être obtenue par cette API** : les micro-entreprises sont de fait toujours exclues du périmètre de cette API. Cet [article d'economie.gouv.fr](https://www.economie.gouv.fr/entreprises/impot-revenu-impot-societe-statut) dresse les types d'imposition possibles par statuts d'entreprise.
```

Reply:
@Haelle j'ai reparlé avec @Charlottecho on est en train d'investiguer, j'avais pas bien compris l'exemple de l'utilisateur qui nous a contacté.

Reply:
Le texte que je propose ici ne répond pas à la question de l'utilisateur :
Pourquoi une société en nom collectif soumise à l'IS ne peut récupérer son attestation en ligne ? Pourquoi on lui dit qu'elle est au régime BIC ?

Reply:
Oui en plus dans notre documentation technique DGFIP, il est clairement indiqué que les BIC, BNC, BA sont exclus du périmètre !

Bon comme il faut toujours des exceptions, on en a trouvé, et on a réussi à obtenir des attestations pour des BIC, BNC, BA... et la DGFIP n'a pas su nous répondre. Par contre à partir du siren/siret et les informations de l'INSEE il est impossible de savoir si l'entreprise est soumise à l'IS ou BIC/BNC/BA...